### ISSUE TITLE
Feature request; option to check free text quiz questions

### ISSUE THREAD

**Issue by tijcolem** [turnitin/moodle-plagiarism_turnitin#340]
Hello, 

Thanks for supporting a great plugin.  I really like the option your plugin in vs the Turnitin2 plugin (https://github.com/turnitin/moodle-mod_turnitintooltwo) since it allows you to 
create native Moodle assignment activities vs creating a specialized Tunitin2 type activity. 

I noticed your plugin support the following (see image below).  Have you thought about adding an option to support quizzes? We use a quiz plugin called coderunner that allows grading of programming type quizzes. It would be nice to be able to check for plagiarism on these submitted quizzes, or free text responses within a quiz. 

<img width="851" alt="screen shot 2018-07-03 at 8 42 30 am" src="https://user-images.githubusercontent.com/7087120/42226509-5cfbe936-7e9c-11e8-87a0-6f60d2b6cd43.png">

Tim

**Comment by cwestgarth** [comment]
Hey @tijcolem 

Thank you for the request! 
This is something which we are currently considering for the future development of our Plagiarism Plugin.

Many thanks
Caroline Westgarth

Product Manager, Integrations

**Comment by mhughes2k** [comment]
So any movement on this since July 2018?

**Comment by aspark21** [comment]
@mhughes2k It's in the TII PP but you need to cherry-pick Dan's WIP in core moodle - https://tracker.moodle.org/browse/MDL-32226

**Comment by danmarsden** [comment]
yeah - I'm just waiting on peer review sign-off before I can send through for integration review. - hopefully should land in Moodle 3.11 - sites using older Moodle releases will likely need to cherry-pick that core patch in the mean-time.

But - as Alastair mentions - the Turnitin plugin already supports it, it's just that the core Moodle Quiz activity needs hooks in the right place so the Turnitin plugin can display the config/links in the right places.

**Comment by carl-hostrander** [comment]
Thank you for reporting this.  Because the latest version of the plagiarism plugin is supported for versions of Moodle 4.1 and higher, I am closing this ticket.  However, if you find this issue is occurring with the latest version of the plugin in any of the supported Moodle versions, please create a new ticket and we will address it.
